website/docs/artist_hosts_nuke_tut.md

```diff
@@ -0,0 +1,166 @@
+      - [Pyblish Checkbox](#pyblish-checkbox)
+      - [Pyblish Dialog](#pyblish-dialog)
+  - [Review](#review)
+  - [Render and Publish](#render-and-publish)
+  - [Version-less Render](#version-less-render)
+  - [Fixing Validate Containers](#fixing-validate-containers)
+  - [Fixing Validate Version](#fixing-validate-version)
+
+This QuickStart is just a small introduction to what OpenPype can do for you. It attempts to make an overview for compositing artists, and simplifies processes that are better described in specific parts of the documentation.
+
+## Run Nuke - Shot and Task Context
+OpenPype has to know what shot and task you are working on. You need to run Nuke in context of the task, using Ftrack Action or OpenPype Launcher to select the task and run Nuke.
+
+![Run Nuke From Ftrack](assets/nuke_tut/nuke_RunNukeFtrackAction_p2.png)
+![Run Nuke From Launcher](assets/nuke_tut/nuke_RunNukeLauncher_p2.png)
+
+
+OpenPype menu shows the current context - shot and task.
+
+If you use Ftrack, executing Nuke with context stops your timer, and starts the Ftrack clock on the shot and task you picked.
+
+![Context](assets/nuke_tut/nuke_Context.png)
+
+:::tip Admin Tip - Nuke version
+You can [configure](admin_settings_project_anatomy.md#Attributes) which version(s) will be available for current project in **Studio Settings → Project → Anatomy → Attributes → Applications**
+:::
+
+## Save Nuke script – the Work File
+Use OpenPype - Work files menu to create a new Nuke script. Openpype offers you the preconfigured naming. 
+![Context](assets/nuke_tut/nuke_AssetLoader.png)
+
+Openpype makes initial setup for your Nuke script. It is the same as running [Apply All Settings](artist_hosts_nuke.md#apply-all-settings) from the OpenPype menu.
+Reads frame range and resolution from Avalon database, sets it in Nuke Project Settings
+Creates Viewer node, sets it’s range and indicates handles by In and Out points
+Reads Color settings from the project configuration, and sets it in Nuke Project Settings and Viewer
+Sets project directory in the Nuke Project Settings to the Nuke Script Directory
+
+:::tip Admin Tip - Workfile Naming
+The [workfile naming](admin_settings_project_anatomy#templates) is configured in anatomy, see **Studio Settings → Project → Anatomy → Templates → Work**
+:::
+
+:::tip Admin Tip - Open Workfile
+You can [configure](project_settings/settings_project_nuke#create-first-workfile) Nuke to automatically open the last version, or create a file on startup. See **Studio Settings → Project → Global → Tools → Workfiles**
+:::
+
+:::tip Admin Tip - Nuke Color Settings
+[Color setting](project_settings/settings_project_nuke) for Nuke can be found in **Studio Settings → Project → Anatomy → Color Management and Output Formats → Nuke**
+:::
+
+## Load plate – Asset Loader
+If your IO or editorial prepared plates and references, or your CG team has a render to be composited, we need to load it.
```

Comment:
It feels like slang words here - we could be more generic here like - data had been prepared for you from upstream processes (rendering, editorial, mediaio).

Reply:
Actually I think that Jirka's version this is clearer and more humanly readable. "Upstream process" is a shiver inducing phrase for artists.

Reply:
I will rephrase